app/Http/Controllers/TransactionController.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Models\transaction;
+use PDF;
+
+class TransactionController extends Controller
+{
+    public function index(){
+        return view('dashboard.createTransaction');
+    }
+
+    public function show(){
+        return view('dashboard.transactionList');
+    }
+
+    public function save(Request $request){
+        $request->validate([
+            'date'=>'required',
+            'description'=>'required',
+            'paid'=>'required',
+            'unit_amount'=>'required|max:10',
+            'unit_quantity'=>'required|max:10',
+            'unit_name'=>'required',
+            'type'=>'required',
+            'status'=>'required',
+            'utr'=>'required',
+            'comments'=>'required',
+            'project'=>'required'
+            ]);
+            $Transaction= new Transaction();
+            $Transaction->date = $request->date;
+            $Transaction->description = $request->description;
+            $Transaction->paid = $request->paid;
+            $Transaction->unit_amount = $request->unit_amount;
+            $Transaction->unit_quantity = $request->unit_quantity ;
+            $Transaction->unit_name = $request->unit_name;
+            $Transaction->type = $request->type;
+            $Transaction->status = $request->status;
+            $Transaction->utr = $request->utr;
+            $Transaction->comments = $request->comments;
+            $Transaction->project = $request->project;
+            $res= $Transaction-> save();
```

Comment:
```suggestion
            $transaction = Transaction::create($request->validated());
```
and use Request Class for validation rule.

Reply:
similar changes throughout.

Reply:
Thanks for the feedback sir, I'll update code in the latest commit.